obf_creation/rendering_types.xml

```diff
@@ -5637,6 +5638,33 @@
 		<routing_type tag="waterway" value="waterfall" mode="register" base="true"/>
 		<routing_type tag="waterway" value="fairway" mode="register" base="true"/>
 		<routing_type tag="seamark:type" value="fairway" mode="register" base="true" />
+		
+		
+		<routing_type tag="aerialway" value="cable_car" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="gondola" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="chair_lift" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="mixed_lift" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="t-bar" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="j-bar" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="platter" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="rope-tow" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="magic_carpet" mode="register" base="true"/>
+		<routing_type tag="aerialway" value="station" mode="register" base="true"/>
+
+
+		<routing_type tag="piste:type" value="downhill" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="connection" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="hike" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="nordic" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="skitour" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="sled" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="snow_park" mode="register" base="true"/>
```

Comment:
This block can be simplified to 
'''xml
<routing_type tag="piste:type" mode="register" base="true"/>
'''